docs/protocol/extension_schedule_policy.md

```diff
@@ -55,6 +57,24 @@ policy. The parameters and their type are:
   the time Triton will take a model-specific action such as
   terminating the request.
 
-Both parameters are optional and if not specified Triton will handle
+Both parameters are optional and, if not specified, Triton will handle
 the request using the default priority and timeout values appropriate
 for the model.
+
+## Sequence Batcher with Direct Scheduling Strategy
+
+**Note that the schedule policy for sequence batcher is at experimental stage
+and it is subject to change.**
+
+The schedule-policy extension uses request parameters to indicate the
+policy. The parameters and their type are:
+
+- "timeout" : int64 value indicating the timeout value for the
+  request, in microseconds. If the request cannot be completed within
+  the time Triton will terminate the request, as well as the corresponding
+  sequence and received requests of the sequence. The timeout will only be
```

Comment:
I think instead of "waiting for ...", it can be changed to "haven't been allocated a batch slot for execution".
"Allocated a batch slot" is the wording used in direct sequence batching [doc](https://github.com/triton-inference-server/server/blob/gluo-seq/docs/user_guide/architecture.md#direct), and it describes the condition on whether the sequence is in backlog:
> Recognizes when an inference request starts a new sequence and allocates a batch slot for that sequence. If no batch slot is available for the new sequence, Triton places the inference request in a backlog.

RE: "will not respect" -> "will not be affected by"?